download.sh

```diff
@@ -13,6 +13,7 @@ N_SHARD_DICT["30B"]="3"
 N_SHARD_DICT["65B"]="7"
 
 echo "Downloading tokenizer"
+mkdir -p ${TARGET_FOLDER}
```

Comment:
Heey!
Would not be better if first checks if that folder exists?

Reply:
Like the following:
```sh
if [[ -f ${TARGET_FOLDER} ]]; then
    echo "${TARGET_FOLDER} exists on your filesystem."
else
    mkdir -p ${TARGET_FOLDER}
```

Reply:
`mkdir -p` is a common pattern for creating a directory if it doesn't already exist.  The -p parameter creates the specified directory (and intermediate parent directories) if they do not exist.

Are there any supported target environments where this does not hold true?
Is there some other advantage to wrapping `mkdir -p` in an if statement?